hack/validate-default-apps-versions.sh

```diff
@@ -1,7 +1,7 @@
 #!/usr/bin/env bash
 set -euo pipefail
 
-latestKommanderVersion=$(find "services/kommander" -maxdepth 1 -type d -regextype sed -regex '.*/[0-9]\+.[0-9]\+.[0-9]\+' -printf "%f\n" | sort --version-sort | tail -1)
+latestKommanderVersion=$(gfind "services/kommander" -maxdepth 1 -type d -regextype sed -regex '.*/[0-9]\+.[0-9]\+.[0-9]\+' -printf "%f\n" | sort --version-sort | tail -1)
```

Comment:
the image used in CI won't have `gfind` unfortunately, which is why pre-commit is failing. 

you can get pre-commit working locally with:
1. `brew install findutils`
2. then add this in your `.zshrc` file:
```
PATH=$(brew --prefix)/opt/findutils/libexec/gnubin:$PATH
```

then source it in your shell and it should be good to go!